rapyuta_io_sdk_v2/utils.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+# Copyright 2024 Rapyuta Robotics
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# from rapyuta_io_sdk_v2.config import Configuration
+import http
+import json
+from typing import Any, Dict
+
+import httpx
+from munch import Munch
+
+from rapyuta_io_sdk_v2.exceptions import HttpAlreadyExistsError, HttpNotFoundError
+
+
+def munchify_projects(response: httpx.Response) -> Munch:
+    data = response.json()
+    projects = []
+    for item in data.get("items", []):
+        project = item.get("metadata", {}).get("name")
+        project_guid = item.get("metadata", {}).get("projectGUID")
+        if project and project_guid:
+            projects.append({"project": project, "project_guid": project_guid})
+    return Munch({"projects": projects})
```

Comment:
question: Why do this? Simply `munchify` the API response as-is. We don't need to modify the response at this time.

Reply:
This is to extarct only relevant data from response it is only used in list_projects.